apps/fluent-tester/src/RNTester/TestComponents/Svg/SvgTest.tsx

```diff
@@ -10,7 +10,6 @@ const styles = StyleSheet.create({
   }
 });
 
-import TestSvg from './Assets/accessible-icon-brands.svg';
```

Comment:
should the asset be removed as well?

Reply:
I'm waiting on @warren-ms to see whether he has a quick fix - if he does I'm just going to drop this entire pr :)

Reply:
Is this blocking something? The fix itself should be quick, just finding it and going through all this build plumbing is taking some time.

Reply:
@samuelfreiberg wanted to enable an e2e test for the svg page. I think he's worked around it, so not needed imminently

Reply:
Not blocked - I have created the tests needed for SVG, I simply haven't added it to the spec document. I also have a task open to remind myself to add it once it's fixed. No rush from me